README.md

```diff
@@ -1,31 +1,303 @@
+- `amount` - amount as a 2.dp decimal number, such as `"1.00"`
+- `show_tips` - (optional) whether to show (true) or hide (false) the tip options. Default is (false)
+
+
+#### PayResponse : Payment Response
+
+````python
+class PaymentStatus(Enum):
+    PENDING = 1
+    SUCCESS = 2
+    FAILED = 3
+    CANCELLED = 4
+
+@dataclass
+class PayResponse:
+    status: PaymentStatus
+    order_id: str  # Unique order ID generated by Kody
+    failure_reason: Optional[str] = None  # Optional, only populated on failure
+    receipt_json: Optional[str] = None  # Optional, json blob for receipt data
+    date_created: datetime  # Timestamp when the response was created
+    ext_payment_ref: Optional[str] = None  # Optional external payment reference
+    date_paid: Optional[datetime] = None  # Optional timestamp for date paid
+    total_amount: Optional[str] = None  # Optional total amount
+    sale_amount: Optional[str] = None  # Optional sale amount
+    tips_amount: Optional[str] = None  # Optional tips amount
+````
+
+#### Python Demo 
+````python
+import kody_clientsdk_python.pay.v1.pay_pb2 as kody_model
+import kody_clientsdk_python.pay.v1.pay_pb2_grpc as kody_client
+
+channel = grpc.secure_channel("https://grpc-staging.kodypay.com", grpc.ssl_channel_credentials())
+kody_service = kody_client.KodyPayTerminalServiceStub(channel)
+metadata = [("x-api-key", "API KEY")]
+
+# Make the client call with request and metadata
+payment_request = kody_model.PayRequest(store_id="STORE ID", 
+                                        terminal_id="TERMINAL ID", 
+                                        amount="65.50",
+                                        show_tips=bool(False))
+payment_response = kody_service.Pay(payment_request, metadata=metadata])
+````
+
+### Get Terminal Payment Details
+
+The payment details request requires the following parameters:
+- `store_id` - the ID of your assigned store
+- `order_id` - the Order ID returned in the initial payment response, a unique UUID value for each payment.
+
+````python
+payment_details_response = kody_client.PaymentDetails(
+    kody_model.PaymentDetailsRequest(store_id="STORE ID", 
+                                     order_id="ORDER ID"),
+    metadata=metadata
+)
+````
+- PayResponse : [Get Payment Detail Response](#payresponse--payment-response)
+
+### Cancel Terminal Payment
+
+#### CancelRequest
+The cancel payment request requires the following parameters:
+
+- `store_id` - the ID of your assigned store
+- `terminal_id` - the serial number of the terminal that is processing the payment request
+- `amount` - the amount sent in the original payment request, used to find the payment request
+- `order_id` - the Order ID returned in the initial payment response, a unique UUID value for each payment
+
+The request authenticates with the server using the `x-api-key` metadata field, you must use your assigned API key.
```

Comment:
I forgot to remove this line. There is no need to mention again how to authenticate the request.
```suggestion

```